compose-builder/add-security.yml

```diff
@@ -187,9 +188,8 @@ services:
       - edgex-network
     ports:
         - "8000:8000"
-        - "127.0.0.1:8001:8001"
         - "8443:8443"
-        - "127.0.0.1:8444:8444"
+        - "8100:8100"
```

Comment:
Does the status port really need to be exposed externally?  It is sufficient to just expose it on the docker network or just expose it to localhost?

```suggestion
        - "127.0.0.1:8100:8100"
```

Reply:
I'll have to test this to see if it will work during the proxy setup process...

Reply:
That worked. Publishing update...